storybook/storybook-docs/src/color.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { Meta } from "@storybook/blocks";
+import { ColourPalette } from "./components/ColourPalette";
+
+<Meta title="Docs/Assets/Colour" />
+
+# Colour
+
+The basic colours of Amsterdam are white, red, and black.
+We have 8 additional colours and 3 shades of grey.
+
+<ColourPalette>
+  <ColourPalette.Section>
+    <ColourPalette.Tile name="white" rgb="#ffffff" />
+    <ColourPalette.Tile name="red" rgb="#ec0000" />
+    <ColourPalette.Tile name="black" rgb="#000000" />
```

Comment:
If we're keeping the custom component: maybe change the prop name from `rgb` to `color`?

Reply:
I’ll be making a case to move to [HSL or even fancier models](https://www.smashingmagazine.com/2021/11/guide-modern-css-colors/) when the time is right. But if we do, we should probably still display the RGB notation as well. I wanted to keep the options open. Should we change to `color` until then?

Reply:
Yeah, I think `color` is a more robust api. We could even allow all types of color notation in the `color` prop, and do any conversion necessary in the component itself... But that's something for later.

Reply:
Alright, done.